session/txn.go

```diff
@@ -391,7 +433,7 @@ func (txn *LazyTxn) LockKeys(ctx context.Context, lockCtx *kv.LockCtx, keys ...k
 	var originState txninfo.TxnRunningState
 	txn.mu.Lock()
 	originState = txn.mu.TxnInfo.State
-	txn.mu.TxnInfo.State = txninfo.TxnLockWaiting
+	txn.updateState(txninfo.TxnLockWaiting)
```

Comment:
Does it count lock waiting status not accurately? Although it's not caused by this PR.
"Lockeys" doesn't mean entering lock wait status.

Reply:
> Does it count lock waiting status not accurately? Although it's not caused by this PR. "Lockeys" doesn't mean entering lock wait status.

Maybe "acquiring locks" would be a better name.